src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter(man => man.sex === 'm'
+    && Math.ceil(man.died / 100) === century)
+    : people.filter(man => man.sex === 'm');
```

Comment:
Краще написати один фільтр і в ньому вибирати колбек
```suggestion
 const man = people.filter(
     сentury 
       ?  person => person.sex === 'm' && Math.ceil(person.died / 100) === century
       : person => person.sex === 'm'
  );
```